bcipy/tasks/tests/test_main_frame.py

```diff
@@ -30,10 +30,15 @@ def test_decide_without_commit(self):
         #print(self.decision_maker.list_epoch[-1]['list_distribution'][-1])
         #print(probability_distribution)
         self.assertFalse(decision)
+        """Test do_epoch method"""
```

Comment:
I would leave this as a separate test. A good rule of thumb is one method or one assertion per unittest. Let's remove the print statement as well (I know you didn't add it!)

Reply:
This should still be removed. Can you remove the commented out print statements above as well?